sourcegraph/sourcegraph
# App: Update application icons for Cody rebrand

Change the app's application icon for the Cody rebrand. This is the icon shown in the dock, on the application file,  and in the About window.

Note that this doesn't change the tray icon.

Part of:

- #53120 

## Test plan

- Run the app
- Check the icon
